src/org/zaproxy/zap/extension/help/ExtensionHelp.java

```diff
@@ -314,6 +321,32 @@ private ZapMenuItem getMenuHelpZapUserGuide() {
 		return menuHelpZap;
 	}
 	
+	private ZapMenuItem getMenuHelpZapHome() {
+		if (menuHelpZapHome == null) {
+			final Object[] options = {"Open", "Ok"};
+			menuHelpZapHome = new ZapMenuItem("help.menu.zap.home");
+			menuHelpZapHome.addActionListener(new java.awt.event.ActionListener() { 
+				@Override
+				public void actionPerformed(java.awt.event.ActionEvent e) {
+					JTextField zapHomeTextField = new JTextField(Constant.getZapHome());
+					zapHomeTextField.setEditable(false);
+					int result = JOptionPane.showOptionDialog(View.getSingleton().getMainFrame(), zapHomeTextField,
+							Constant.messages.getString("help.menu.zap.home.title"), JOptionPane.YES_NO_OPTION, 
+							JOptionPane.PLAIN_MESSAGE, null, options, options[0]);
+					if (result == JOptionPane.YES_OPTION) {
+						try {
+							Desktop.getDesktop().open(new File(Constant.getZapHome()));
```

Comment:
We should check if the action is supported, otherwise there's no need to show the button (it would fail to open the dir always).